modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/internal/upgrade/RemoteAppServiceUpgrade.java

```diff
@@ -35,6 +35,11 @@ public void register(Registry registry) {
 			"1.0.1", "2.0.0",
 			new com.liferay.remote.app.internal.upgrade.v2_0_0.
 				RemoteAppEntryUpgradeProcess());
+
+		registry.register(
```

Comment:
Usually, each upgrade class refers only to one table or entity, that is why I created a new upgrade, although it would have been better to increase the hotfix level instead of the minor level